accounts-db/src/ancient_append_vecs.rs

```diff
@@ -522,9 +516,16 @@ impl AccountsDb {
     fn collect_sort_filter_ancient_slots(
         &self,
         slots: Vec<Slot>,
-        tuning: &PackedAncientStorageTuning,
+        tuning: &mut PackedAncientStorageTuning,
     ) -> AncientSlotInfos {
         let mut ancient_slot_infos = self.calc_ancient_slot_info(slots, tuning);
+        // ideal storage size is total alive bytes of ancient storages
```

Comment:
Can we add a test for this, i.e. after we call collect_sort_filter_ancinet_slots() make sure that ideal_storage_size is updated?